packages/hash/api/src/graph/ontology/primitive/data-type.ts

```diff
@@ -29,7 +29,7 @@ import { NotFoundError } from "../../../lib/error";
  */
 export const createDataType: ImpureGraphFunction<
   {
-    ownedById: OwnedById;
+    ownedById: AccountId;
```

Comment:
I made this pass in the last commit since I would expect us to allow any `AccountId` to be the owner of the ontology types. Otherwise we'd have to `brand` to upcast at the GQL resolvers to turn the `AccountId` into an `OwnedById`.
The types that inherit from `AccountId` such as `OwnedById` would make most sense to be returning from the subgraph, but for creation I suppose we want to be as permissive as possible as an `UpdatedById` is still a valid `ownedById` to pass into this function

Reply:
Right but it's the whole point to require branding for that 😅, especially for this:

> but for creation I suppose we want to be as permissive as possible as an UpdatedById is still a valid ownedById to pass into this function

That's the one where we wanted to absolutely require it to be explicit, because we don't want us mixing up IDs so we require you to go "I know this is an `UpdatedById`, I'm going to turn it into an `OwnedById`

Reply:
2e79e039d40883c9728dd67f49e50266b1d43ebf Hopefully that should be all. I need to use a mix of `brand` and `as` because of `null` choices. I wonder if we want to/can make that more ergonomic?